### ISSUE TITLE
The modal is tricky to close

### ISSUE THREAD

**Issue by vinyll** [productownerinTO/producTOwner#3]
It took me a short while to succeed closing the modal window (only the top left icon – which is currently not displayed – closes the modal).
I'd expect the modal to close when clicking outside of it or when pressing the _esc_ key.

**Comment by fuentesloic** [comment]
Ok so for that I need JS right ?

**Comment by vinyll** [comment]
Yes. Maybe you should check the documentation from the modal lib you are using.
